x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -251,24 +251,36 @@ export const getManagementFilteredLinks = async (
           endpointPermissions
         )
       : getEndpointAuthzInitialState();
-    if (!privileges.canAccessEndpointManagement) {
-      return excludeLinks([
-        SecurityPageName.hostIsolationExceptions,
-        SecurityPageName.responseActionsHistory,
-      ]);
+
+    // exclude links based on privileges
+    if (!canAccessEndpointManagement || !canReadActionsLogManagement) {
```

Comment:
(here and below) I don't think we should be using `canAccessEndpointManagement` anymore, right?

Reply:
Should we use `canReadEndpointList` here instead or just not use this check at all?